Fakebook.Posts/Fakebook.Posts.IntegrationTests/Fakebook.Posts.IntegrationTests.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
projects doing asp.net core integration tests should have the test sdk up here:
https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-5.0#test-app-prerequisites